src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,138 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
Instead of AUTO we use SEQUENCE because PostgreSQL uses a global incremental id, shared by all tables :)

Reply:
great, thanks